test/gtest/common/test_helpers.cc

```diff
@@ -15,6 +15,94 @@ namespace ucs {
 
 const double test_timeout_in_sec = 60.;
 
+const double watchdog_timeout_default = 900.; // 15 minutes
+
+static test_watchdog_t watchdog;
+
+void *watchdog_func(void *arg)
+{
+    int ret = 0;
+    struct timeval now;
```

Comment:
plz consider using uct_time_t instead

Reply:
good catch. done - replaced with `ucs_get_accurate_time()`, it because doesn't work with `usc_get_time()`. Looks like we need to use the same time getter as `pthread` uses